addons/mail/static/tests/owl/component/discuss_tests.js

```diff
@@ -3805,29 +3918,7 @@ QUnit.test('composer state: text save and restore', async function (assert) {
 });
 
 QUnit.test('composer state: attachments save and restore', async function (assert) {
```

Comment:
I would drop this test, and test each messaging actions instead

Reply:
I don't understand why ? I think it's great to check that UI is really "re-mounted" in the good state with all attachments, no ?

Reply:
Yes for checking the UI, no for checking store state.
I thought there was already a test for checking the UI, but apparently no.
So if store state checks are replaced by `querySelectorAll()`, then it's good :)